resources/wikia/modules/pageviewsInSession.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Pageviews in session counter
+ *
+ * What is definition of session in this case?
+ * Session is series of page views (across one or many wikis) done by one user.
+ * It starts when user first enter wiki page and ends when user close last browser tab with wiki page.
+ *
+ * One case is not covered in this solution:
+ * User is on wiki page, navigate to external service and then type wiki address into browser address bar.
+ * When the url is typed in this bar we don't have information about history and last page.
+ */
+define('wikia.pageviewsInSession',
+	['wikia.window', 'wikia.document', 'wikia.cookies', 'mw'], function (window, document, cookies, mw) {
+	'use strict';
+
+	var wikiaDomain = mw.config.get('wgDevelEnvironment') ? '.wikia-dev.com' : '.wikia.com',
+		pageviewsCookieName = 'pageviewsInSession',
+		opentabsNumberCookieName = 'openTabsNumber';
+
+	function init() {
+		increaseNumberOfOpenTabs();
+		setPageviewsCount();
+
+		window.addEventListener('unload', function () {
+			decreaseNumberOfOpenTabs();
+			sessionStorage.setItem('wasWikiPageShownInCurrentTab', true);
```

Comment:
I'd suggest either accessing this global on the window object, or creating a small AMD module like our [one for localStorage](https://github.com/Wikia/app/blob/dev/resources/wikia/modules/localStorage.js).